examples/FUOTA/UpdateServer/MQTT/ChirpStackMQTT.py

```
@@ -0,0 +1,46 @@
+        self.__Client = mqtt.Client()
+        self.__Client.on_connect = self.__on_MQTT_Connect
+        self.__Client.on_message = self.__on_MQTT_Message
+        self.__Client.connect(Server, Port, 60)
+        self.__Client.loop_start()
+
+        self.__Status = -1
+
+    def __del__(self):
+        self.__Client.loop_stop()
+
+    def __on_MQTT_Connect(self, Client, UserData, Flags, ReturnCode):
```

Ruff (0.14.3)

Unused method argument: `Client`

(ARG002)